src/components/product.js

```diff
@@ -52,6 +52,22 @@ function whitelistedProperties(selectorStyles) {
   }, {});
 }
 
+function adaptConfig(config) {
```

Comment:
this isn't doing anything with a config. It's normalizing an ID. Consider a different name.